crawlers/crawlDomain.js

```diff
@@ -101,17 +110,51 @@ const crawlDomain = async (
       if (isBasicAuth) {
         isBasicAuth = false;
       } else if (location.host.includes(host)) {
-        const results = await runAxeScript(page);
+        const results = await runAxeScript(needsReview, page);
+
+        // For deduplication, if the URL is redirected, we want to store the original URL and the redirected URL (actualUrl)
+        const isRedirected = !areLinksEqual(request.loadedUrl, request.url);
+        if (isRedirected) {
+          const isLoadedUrlInCrawledUrls = urlsCrawled.scanned.some(item => {
+            (item.actualUrl || item.url) === request.loadedUrl;
+          });
```

Comment:
```suggestion
          const isLoadedUrlInCrawledUrls = urlsCrawled.scanned.some(item => 
            (item.actualUrl || item.url) === request.loadedUrl;
          );
```
if there is the curly braces, believe nothing is returned